src/components/v5/common/CompletedAction/partials/StreamingPayment/consts.ts

```diff
@@ -0,0 +1,2 @@
+export const ONE_HOUR_IN_SECONDS = 3600;
+export const ONE_DAY_IN_SECONDS = ONE_HOUR_IN_SECONDS * 24;
```

Comment:
These exist in 2 places now, maybe we can just put these in `src/constants` as I imagine they'll be useful elsewhere too.

<img width="604" alt="Screenshot 2024-09-05 at 11 27 08" src="https://github.com/user-attachments/assets/937a370f-ff76-4f14-bcc0-52ad62fe4ee3">
<img width="603" alt="Screenshot 2024-09-05 at 11 27 59" src="https://github.com/user-attachments/assets/7cea2c06-c2f9-4696-8f16-c7efed67f7af">